src/Acuminator/Acuminator.Vsix/Resources/VSIXResource.resx

```diff
@@ -264,4 +264,22 @@
   <data name="CodeMapWindowTitle" xml:space="preserve">
     <value>Code Map (Experimental)</value>
   </data>
+  <data name="PackageLoad_Done" xml:space="preserve">
```

Comment:
Were these strings reviewed?

Reply:
No, I also can’t find them in UI. But I agree, I’ll ask kseniya for review

Reply:
@k-s-popova please review strings added in this file

Reply:
The strings were reviewed